src/windows/process.rs

```diff
@@ -599,6 +599,10 @@ impl ProcessExt for Process {
         self.cpu_usage
     }
 
+    fn total_cpu_usage(&self) -> f32 {
```

Comment:
I have absolutely no idea if you can get this value on Windows. Maybe a query allows to get this information. You can check how CPU is updated in `windows/system.rs::refresh_cpu_specifics`